### PR TITLE
fix: adapt to checkstyle 8.29

### PR DESCRIPTION
**Pull request by lpetit-yseop**

### CODE REVIEW

File: java/codeChecks/Yseop_Code_Checks.xml [hunk 1]
```diff
@@ -180,12 +180,9 @@
       <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
     </module>
     <module name="JavadocMethod">
-      <property name="minLineCount" value="2"/>
```
Comment (eborel-yseop) [inline java/codeChecks/Yseop_Code_Checks.xml]: Looking at the properties name they do not seems to be that important, or maybe they're default now ? Anyway not blocking for me.

**Comment by mminot-yseop** [comment]
Oh nice. I suppose this kinda matches what Gupta had to do to make it work? Actually, I think it was crashing on my machine too, and I just didn’t notice until he mentionned these issues.

File: java/codeChecks/Yseop_Code_Checks.xml (hunk 1 above)
Comment (lpetit-yseop) [reply java/codeChecks/Yseop_Code_Checks.xml]: Extract from the commit message:

> - minLineCount is not used anymore, see checkstyle/checkstyle#7096
> - allowMissingThrowsTags and allowThrowsTagsForSubclasses are not used anymore, see checkstyle/checkstyle#7329

Comment (eborel-yseop) [reply java/codeChecks/Yseop_Code_Checks.xml]: :ok: :+1:

**Comment by mminot-yseop** [comment]
Hum I still get

> Checkstyle execution failed due to an internal error. Please check the error log for details...

I don’t even know where those logs are. :sweat_smile:
As far as I remember, if the formater is not changed, no action in Eclipse prefs is required once the local code quality clone is updated. :thinking: Not sure what I’m doing wrong.

**Comment by mminot-yseop** [comment]
> Hum I still get
> 
> > Checkstyle execution failed due to an internal error. Please check the error log for details...
> 
> I don’t even know where those logs are. sweat_smile
> As far as I remember, if the formater is not changed, no action in Eclipse prefs is required once the local code quality clone is updated. thinking Not sure what I’m doing wrong.

Hum seems to be gone now :woman_shrugging:

**Pull request merged**